drivers/pwm/pwm_mcux_ftm.c

```diff
@@ -27,12 +32,27 @@ struct mcux_ftm_config {
 	ftm_clock_prescale_t prescale;
 	uint8_t channel_count;
 	ftm_pwm_mode_t mode;
+#ifdef CONFIG_PWM_CAPTURE_API
+	void (*irq_config_func)(const struct device *dev);
+#endif /* CONFIG_PWM_CAPTURE_API */
+};
+
+struct mcux_ftm_capture_data {
+	ftm_dual_edge_capture_param_t param;
+	pwm_capture_callback_handler_t callback;
+	void *user_data;
+	uint32_t first_edge_overflows;
+	bool pulse_capture : 1;
```

Comment:
Why use bitfield syntax here?

Reply:
That was a leftover from an earlier iteration where I had more boolean values in the same struct. Fixed.